e2e/test/cli_test.go

```diff
@@ -0,0 +1,138 @@
+package e2e_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/omni-network/omni/cli/cmd"
+	"github.com/omni-network/omni/lib/anvil"
+	"github.com/omni-network/omni/lib/cchain/provider"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/lib/xchain"
+
+	"github.com/cometbft/cometbft/rpc/client/http"
+
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/stretchr/testify/require"
+)
+
+// exeCLI will execute provided command with the arguments and return an error in case
+// execution fails, or command output as string in case of success.
+func exeCLI(args ...string) (string, error) {
+	outBuf := new(bytes.Buffer)
+	errBuf := new(bytes.Buffer)
+
+	root := cmd.New()
+	root.SetOut(outBuf)
+	root.SetErr(errBuf)
+
+	root.SetArgs(args)
+	if err := root.Execute(); err != nil {
+		return "", err
+	}
+	if errBuf.Len() > 0 {
+		return "", errors.New(errBuf.String())
+	}
+
+	return outBuf.String(), nil
+}
+
+// TestValidatorCommands tests multiple CLI operator commands in sequence.
+// The test runs the following commands:
+// - operator create-validator creates a new validator and makes sure the validator is added to the consensus chain
+// - operator delegate increases the newly created validator stake and makes sure its power is increased
+//
+// Since they rely first on validator being created it must be run as a unit.
+func TestValidatorCommands(t *testing.T) {
+	t.Parallel()
+
+	testNetwork(t, func(t *testing.T, network netconf.Network, endpoints xchain.RPCEndpoints) {
+		t.Helper()
+		testnet, _, _, _ := loadEnv(t)
+
+		e, ok := network.OmniEVMChain()
+		require.True(t, ok)
+		executionRPC, err := endpoints.ByNameOrID(e.Name, e.ID)
+		require.NoError(t, err)
+
+		// use an existing test anvil account for new validator and write it's pkey to temp file
+		validatorPriv := anvil.DevPrivateKey6()
+		validatorPub := ethcrypto.CompressPubkey(&validatorPriv.PublicKey)
```

Comment:
rather indicate if something is bytes by adding a `bz` suffix (cometBFT style)

Reply:
what is bz?

Reply:
`bz` means `bytes`. It is cometBFT and cosmosSDK style